ingest/expression_files/mtx.py

```diff
@@ -65,7 +63,11 @@ def check_valid(
         except ValueError as v:
             error_messages.append(str(v))
         try:
-            MTXIngestor.check_duplicate_genes(genes)
+            MTXIngestor.check_duplicates(genes, "gene")
+        except ValueError as v:
+            error_messages.append(str(v))
+        try:
+            MTXIngestor.check_duplicates(barcodes, "barcodes")
```

Comment:
Singular, as done for "gene": 

```suggestion
            MTXIngestor.check_duplicates(barcodes, "barcode")
```